content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -9,7 +9,7 @@ weight: 90
 ---
 
 {{% capture overview %}}
-A service account provides an identity for processes that run in a Pod.
+A _service account_ provides an identity for processes that run in a Pod.
```

Comment:
Service account vs service account?

Reply:
I think “service account” is fine for English grammar. The Kubernetes resource is ServiceAccount, which gets introduced below.

Reply:
Do you need italics? Check the style guide.

Reply:
Tweaked